### PR TITLE
chore(L10n): French Translation Update

### PR DESCRIPTION
**Pull request by Luc-Webster**
A few new contents added

### CODE REVIEW

File: phira/locales/fr-FR/common.ftl
Comment (sjfhsjfh) [inline phira/locales/fr-FR/common.ftl:1]: Maybe merge after this translation?

Comment (YuevUwU) [reply phira/locales/fr-FR/common.ftl:1]: ## TODO Translation Report for fr-FR

_Generated by bce7291da5ce98ab3f9627bdfe191945fe30038d_
_**NOTICE: This report may not cover all targets to be translated**_

### phira/locales/fr-FR/

#### Append needed

[common.ftl](https://github.com/TeamFlos/phira/tree/main/phira/locales/en-US/common.ftl) - tos-prev-page  
[common.ftl](https://github.com/TeamFlos/phira/tree/main/phira/locales/en-US/common.ftl) - tos-next-page  
[home.ftl](https://github.com/TeamFlos/phira/tree/main/phira/locales/en-US/home.ftl) - note-try-login-again  
[home.ftl](https://github.com/TeamFlos/phira/tree/main/phira/locales/en-US/home.ftl) - change-char  

#### Edit needed

```plaintext
multiplayer.ftl - msg-game-start  
en-US: The host `{ $user }` started the game. Other players should get ready.  
fr-FR: L'hôte(sse) `{ $user }` a démarré(e) le jeu  
```

_**Thanks for the trouble**_

**Comment by sjfhsjfh** [comment]
I'd like to keep this open, maybe a fix in 0.7.0

**Comment by Luc-Webster** [comment]
Now they're basically complete. Feel free to report any further missing strings or mistranslation.

**Review by sjfhsjfh — APPROVED** [review]

**Pull request merged**